server/auth/types/basic/routes.ts

```diff
@@ -35,7 +35,7 @@ export class BasicAuthRoutes {
     private readonly coreSetup: CoreSetup
   ) {}
 
-  public async setupRoutes() {
+  public setupRoutes() {
```

Comment:
Just wondering whether this is intentional

Reply:
yes, I used to have a async call in this function, but it is removed now